launch/launch/actions/execute_local.py

```diff
@@ -199,7 +200,17 @@ def __init__(
             self.__output = normalize_to_list_of_substitutions(tmp_output)
         else:
             self.__output = tmp_output
-        self.__output_format = output_format
+
+        # We use the following priorities to determine the output_format:
+        # 1. Passed value to the function
+        # 2. Environment variable
+        # 3. Default value
+        if output_format != "[{this.process_description.final_name}] {line}":
```

Comment:
I dislike that this string literal, `[{this.process_description.final_name}] {line}`, is in two places, I would ask that you either change this to default to `None` and check for that here, or make a separate constant with the string literal and use that in the two places you need it.

I prefer the first approach, because if I do `ExecuteLocal(..., output_format="[{this.process_description.final_name}] {line}", ...)` I would expect that it overrides the env var, despite it being the same as the default value, because I explicitly passed it.

Reply:
Good point, I changed the code so that providing the default format via the argument will lead to this being used. This also resolved the double string literal issue.